otx/api/usecases/exportable_code/demo/requirements.txt

```diff
@@ -1,2 +1,3 @@
 openmodelzoo-modelapi==2022.2.0
 otx @ git+https://github.com/openvinotoolkit/training_extensions/@e12437e6660b257731e76192344d01c70f603f73#egg=otx
+numpy<=1.23.5  # np.bool was removed in 1.24.0 which was used in openvino runtime
```

Comment:
How about using numpy>=1.21.0,<=1.23.5 ? Like same as below figure
![image](https://user-images.githubusercontent.com/80735344/208805417-4f7e437e-6362-4275-843b-835f42107639.png)

Reply:
Does this conflict with the NNCF and OpenVINO installation? I'm getting errors during the installation which following the quick start guide.

ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
openvino 2022.1.0 requires numpy<1.20,>=1.16.6, but you have numpy 1.24.0 which is incompatible.
openvino-dev 2022.1.0 requires numpy<1.20,>=1.16.6, but you have numpy 1.24.0 which is incompatible.
openvino-dev 2022.1.0 requires numpy<=1.21,>=1.16.6; python_version > "3.6", but you have numpy 1.24.0 which is incompatible.
openmodelzoo-modelapi 0.0.0 requires numpy<=1.21,>=1.16.6; python_version > "3.6", but you have numpy 1.24.0 which is incompatible.
nncf 2.1.0.dev0+e85a695d requires numpy~=1.19.2, but you have numpy 1.24.0 which is incompatible.

Reply:
Hello, this is feature/otx branch PR.
You may want to check develop branch PR: https://github.com/openvinotoolkit/training_extensions/pull/1455

Yes, there are some version mismatches that pip is complaining when you run init_venv.sh.
But it will work well actually.

This mismatch will be resolved after we could upgrade the OpenVINO version by https://github.com/openvinotoolkit/training_extensions/pull/1393